pkg/bootstrap/boostrapkubeconfig.go

```diff
@@ -184,6 +193,11 @@ func GetBootstrapCAData(ctx context.Context, clientHolder *helpers.ClientHolder,
 		return klusterletConfig.Spec.HubKubeAPIServerCABundle, nil
 	}
 
+	// get caBundle from the kube-root-ca.crt configmap in the pod namespace for non-ocp case.
```

Comment:
checked in AKS with 1.28.3 k8s,  it has kube-root-ca.crt configmap.
otherwise, we also can configure ca via klustereletconfig as long as the hub apiserver URL.